### PR TITLE
Finished draft of copy and added new screenshots.

### PR DESCRIPTION
**Pull request by Vwampage**

### CODE REVIEW

File: content/guides/ship-with-kubernetes/create-release.md [hunk 1]
```diff
@@ -30,27 +30,13 @@ You'll should be at the channels page now. This is a list of your release channe
 
 {{< linked_headline "Create a Release" >}}
 
-You should now see a YAML editor where you can define how you application will work and the integration with Replicated functionality. Once you are familiar with this concepts, you'll probably use our CLI and API to automate this and not manually edit YAML in this site. But for now, let's continue by adding the Guestbook example.
+You should now see a YAML editor where you can define how you application will work and the integration with Replicated functionality. Once you are familiar with this concepts, you'll probably use our CLI and API to automate this and not manually edit YAML on this page. The default YAML documents contain some boilerplate Replicated settings and no Kubernetes specs. For now let's click the "Save release" button in the bottom right.
 
 ![Default YAML](/images/guides/kubernetes/default-yaml.png)
```
Comment (austinchambers) [inline content/guides/ship-with-kubernetes/create-release.md]: I think we need content here, explaining the default yaml provided with the KOTS application.

File: content/guides/ship-with-kubernetes/create-release.md [hunk 2]
```diff
@@ -14,11 +14,11 @@ icon: "replicatedKubernetes"
 
 When starting and learning the Replicated platform, the [Vendor Portal](https://vendor.replicated.com) will be the place you spend most of your time. This guide is designed to help you get familiar with the concepts and ideas that are important to successfully deploy your application with Replicated. If you get stuck or need help, head to our [community](https://help.replicated.com/community/).
 
-This guide will deploy one of the canonical Kubernetes applications using Replicated, and then iterate on it to deliver an update. The guide isn't going to teach Kubernetes, but will start with a working Kubernetes application ([Guestbook all-in-one example](https://raw.githubusercontent.com/kubernetes/examples/12cff4b7a59334080958469052bf9cde80e75618/guestbook/all-in-one/guestbook-all-in-one.yaml)).
+This guide will deploy a basic application using Replicated and show you how to deliver an update to that application. The guide isn't going to teach Kubernetes, but will start with a working Kubernetes application.
```
Comment (austinchambers) [inline content/guides/ship-with-kubernetes/create-release.md]: I wonder if we should replace this with the URL for our starter-kots-repo?  Or maybe kots-sentry?

File: content/guides/ship-with-kubernetes/create-release.md (hunk 1 above)
Comment (Vwampage) [reply content/guides/ship-with-kubernetes/create-release.md]: I added some more content but would welcome your thoughts on what should/should not be in there.

File: content/guides/ship-with-kubernetes/create-release.md (hunk 2 above)
Comment (dexhorthy) [reply content/guides/ship-with-kubernetes/create-release.md]: This is technically the YAML in replicatedhq/kots-default-yaml, but i would rather link to starter-kots if we link anywhere. (And tbh we should find a way to keep those things in sync)

Comment (dexhorthy) [reply content/guides/ship-with-kubernetes/create-release.md]: Let's save this as a followup task

File: content/guides/ship-with-kubernetes/create-release.md (hunk 1 above)
Comment (dexhorthy) [reply content/guides/ship-with-kubernetes/create-release.md]: lgtm, I think at some point we can add a link to the vendor docs

**Pull request merged**